mathics/builtin/__init__.py

```diff
@@ -6,7 +6,8 @@
     algebra, arithmetic, assignment, attributes, calculus, combinatorial, compilation,
     comparison, control, datentime, diffeqns, evaluation, exptrig, functional,
     graphics, graphics3d,
-    image, inout, integer, iohooks, linalg, lists, logic,
+    # image,
+    inout, integer, iohooks, linalg, lists, logic,
```

Comment:
This right here is what I'm talking about.

Reply:
Yes, my mistake. I always comment out this module for local testings...

Reply:
No probs, nobody's perfect!